### ISSUE TITLE
jupyter slideshow doesn't work

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#2569]
1. Create a jupyter notebook (modern)
2. Click File --> Slides
3. Open the resulting html file (e.g, using the download link to get a nice html viewer)
4. It does not work. The console shows why -- missing reveal.js:

![screenshot 2017-12-11 at 2 56 49 pm](https://user-images.githubusercontent.com/1276278/33858213-aeb5e42a-de83-11e7-84d5-abd7829dedb2.png)

**Comment by williamstein** [comment]
Reported by:  Stefan Lukits

**Comment by williamstein** [comment]
Here is a workaround until we do something more automatic.

1. Make sure your ipynb file is saved.
2. Open a terminal in CoCalc by typing (+New Terminal).  It's best to do this in the same directory as your ipynb file.
3. Type this, where you replace `slides.ipynb` with the name of your ipynb file. 
```
jupyter nbconvert --to slides --ServePostProcessor.port=8080 --ServePostProcessor.ip='*' --ServePostProcessor.open_in_browser=False "slides.ipynb" --post serve
```
4. Open this link in another browser tab:
```
https://cocalc.com/45f4aab5-7698-4ac8-9f63-9fd307401ad7/server/8080/slides.slides.html#/
```
where you replace `45f4aab5-7698-4ac8-9f63-9fd307401ad7` by the id of your project (which in the project settings or the URL bar).

**Comment by williamstein** [comment]
I hope people will test this on complicated slideshows and see if it works well.

**Comment by haraldschilly** [comment]
Similar technology, but more modern: did you look into "[RISE](https://damianavila.github.io/RISE/)". Should we work on creating something similar to it?

I installed it locally in my test project for the classical notebook. It's using the same overall reveal.js technology, but runs right inside the notebook ... i.e. it doesn't use a  server. It also allows you to execute the cells.

```
pip3 install --user rise
jupyter-nbextension install rise --py --user
jupyter-nbextension enable rise --py --user
```

and then there is a new button in the classical notebook interface

**Comment by williamstein** [comment]
> did you look into "RISE". 

Yes.  It's an extension which works *inside* classical Jupyter, and allows for live interaction with the slides.  In other words, it's basically like the slideshow mode that we wrote for sagenb in 2007, but with much nicer Javascript/navigation. 

You should definitely put making RISE available by default on your todo list for software installs, and when I redo this properly (not just a quick hack), I'll want to clone RISE inside CoCalc Jupyter.
